servant/src/main/java/com/mtramin/servant/GoogleApiClientObservable.java

```diff
@@ -45,11 +45,11 @@ private GoogleApiClientObservable(Context context, GoogleApi googleApi) {
 
     static Observable<GoogleApiClient> create(Context context, GoogleApi googleApi) {
         return Observable.fromEmitter(new GoogleApiClientObservable(context, googleApi),
```

Comment:
Awesome, thanks! Will wait for CI to go green and will look at building a new release later this week.